### DISCUSSION TITLE
Ask Snowflake - Couldn't run sql:  'NoneType' object has no attribute 'fetchall'

### DISCUSSION THREAD

**Discussion by dr-qian** [vanna-ai/vanna#272] [Q&A]
I am running vn.ask(question="some natual language questions") to query Snowflake table, and it returns the following error.

None
WARNING:snowflake.connector.cursor:execute: no query is given to execute
Couldn't run sql:  'NoneType' object has no attribute 'fetchall'

**Comment by zainhoda** [comment]
More information about your setup and steps are necessary here. For this particular question, a SQL statement failed to generate but it could be due to any number of reasons without additional information.

Please provide details on how you set up Vanna (preferably showing all the steps leading up to creation of the `vn` object).
